opensource/project/set-up-dev-env.md

```diff
@@ -275,13 +275,17 @@ example, you'll edit the help for the `attach` subcommand.
 
 5. Save and close the `cli/command/container/attach.go` file.
 
-6. Go to your running development container.
+6. Go to your running docker development container shell.
 
-7. Remake the binary and copy it to `usr/bin`
+7. Remake the binary by entering `hack/make.sh binary`  in the docker development container shell.
```

Comment:
Consider "rebuild" or another verb. Maybe something like "using the command" instead of "entering".

Nit: double space before 'in'